crates/ef-testing-proc-macro/src/content_reader.rs

```diff
@@ -0,0 +1,54 @@
+use std::collections::BTreeMap;
+
+use serde_json::Value;
+
+use crate::{dir_reader::PathWrapper, utils::blockchain_tests_to_general_state_tests_path};
+
+pub struct ContentReader;
+
+impl ContentReader {
```

Comment:
Please add rustdoc to express that this content reader is meant for one particular test case, inside a LotsOfTestCases.json file,
give example of structure